src/test/scala/com/cognite/sdk/scala/v1/FilesTest.scala

```diff
@@ -243,4 +256,30 @@ class FilesTest extends SdkTestSpec with ReadBehaviours with WritableBehaviors w
     assert(out.toByteArray().sameElements(expected))
     client.files.deleteById(file.id)
   }
+
+  it should "support search with dataSetIds" in {
+    val created = filesToCreate.map(f => client.files.createOneFromRead(f))
+    try {
+      val fromTime = created.map(_.createdTime).min
```

Comment:
Since this is only used once and the meaning is clear enough, I wouldn't bother introducing a variable for it.
You could however do `val createdTimes = created.map(_.createdTime)` and then `min=createdTimes.min` below.